ddtrace/contrib/internal/django/user.py

```diff
@@ -0,0 +1,82 @@
+from ddtrace.appsec._utils import _UserInfoRetriever
+from ddtrace.internal.logger import get_logger
+from ddtrace.settings.asm import config as asm_config
+
+
+log = get_logger(__name__)
+
+
+class _DjangoUserInfoRetriever(_UserInfoRetriever):
```

Comment:
copy/pasta from `patch.py` needed to move since now multiple modules depend on it.